src/diffpy/utils/tests/test_tools.py

```diff
@@ -46,20 +46,20 @@ def _run_tests(inputs, expected):
     (["", ""], ["input_username", "input@email.com"], ["input_username", "input@email.com"]),
     (["cli_username", ""], ["", "input@email.com"], ["cli_username", "input@email.com"]),
     (["", "cli@email.com"], ["input_username", ""], ["input_username", "cli@email.com"]),
-    (["cli_username", "cli@email.com"], ["cli_username", "cli@email.com"], ["cli_username", "cli@email.com"]),
+    (["cli_username", "cli@email.com"], ["input_username", "input@email.com"], ["cli_username", "cli@email.com"]),
```

Comment:
I think this should be input names rather than cli? And this means that cli overrides inputs?

Reply:
no, cli overrides everything.  A user would only supply this information at runtime if they want it.   We only trigger the input sequence when there is no config file found and nothing specified on the command line for either username, or email or both.

Reply:
oh, wait, I see what you are saying.  Yes, you are right!  Nicely done!